src/app/lib/business/bugsee/bugsee_manager.dart

```diff
@@ -138,62 +212,100 @@ final class _BugseeManager implements BugseeManager {
             "BUGSEE: not initialized, verify bugsee token configuration",
           );
         }
-        _isBugSeeInitialized = isBugseeLaunched;
+        isInitialized = isBugseeLaunched;
       },
       launchOptions: launchOptions,
     );
+    if (configurationData.isLogsFilterEnabled ?? false) {
+      Bugsee.setLogFilter(_filterBugseeLogs);
+    }
+    if (configurationData.attachLogFileEnabled ?? false) {
+      Bugsee.setAttachmentsCallback(_attachLogFile);
+    }
+    return isInitialized;
   }
 
   BugseeLaunchOptions? _initializeLaunchOptions() {
     if (Platform.isAndroid) {
-      return AndroidLaunchOptions();
+      return AndroidLaunchOptions()
+        ..captureLogs = (configurationData.isLogCollectionEnabled ?? true);
     } else if (Platform.isIOS) {
-      return IOSLaunchOptions();
+      return IOSLaunchOptions()
+        ..captureLogs = (configurationData.isLogCollectionEnabled ?? true);
     }
     return null;
   }
 
-  @override
-  Future<void> logException({
-    required Exception exception,
-    StackTrace? stackTrace,
-  }) async {
-    if (isBugseeEnabled) {
-      await Bugsee.logException(exception, stackTrace);
+  Future<BugseeLogEvent> _filterBugseeLogs(BugseeLogEvent logEvent) async {
+    logEvent.text = logEvent.text.replaceAll(RegExp(bugseeFilterRegex), '');
+    return logEvent;
+  }
+
+  Future<List<BugseeAttachment>> _attachLogFile(BugseeReport report) async {
+    var attachments = <BugseeAttachment>[];
+    if (loggerManager.logFile != null) {
+      attachments.add(
+        BugseeAttachment(
+          "logFile",
+          loggerManager.logFile!.path,
+          loggerManager.logFile!.readAsBytesSync(),
+        ),
+      );
     }
+    return attachments;
   }
 
   @override
-  Future<void> logUnhandledException({
+  Future<void> logException({
     required Exception exception,
     StackTrace? stackTrace,
+    Map<String, dynamic>? traces,
```

Comment:
What are the use-cases for the `traces` and `events` parameters? Looking at the code, they don't seem related to the exception, and therefore not related to this method.